config/initializers/content_security_policy.rb

```diff
@@ -6,7 +6,7 @@
 
 Rails.application.configure do
   # Allow disabling CSP in development because it breaks Vue devtools in Firefox.
-  if Rails.env.development? && ENV.key?('DISABLE_CSP')
+  if Rails.env.local? && ENV.key?('DISABLE_CSP')
```

Comment:
I wanted to test that AJAX requests were also subjected to the same domain restrictions as the top level page navigations. (They are.) This required disabling the CSP. This change makes that easier.